.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions? Did you do something clever?
+
+Provide any external context for the change, if any.
+
+For example: Kubernetes links that explain why a change is
+required, or CSI spec related changes that may
+need to be pointed to, or generic golang related practices
+that may need to be pointed to.
```

Comment:
(nits) Reword slightly

"Provide any available external context for the change.

For example:
  - Kubernetes links that explain why the change is required
  - CSI spec related changes/catch-up that necessitates this patch
  - golang related practices that necessitates this change"

Reply:
Done.